lms/lmstests/public/flake8/services.py

```diff
@@ -92,10 +93,12 @@ def _fire_notification_if_needed(self):
 
         errors_len = len(self._errors)
         exercise_name = self.solution.exercise.subject
-        msg = (
-            f'הבודק האוטומטי נתן {errors_len} '
-            f'הערות על תרגילך "{exercise_name}".'
-        )
+        msg = ''.join((
+            gettext('הבודק האוטומטי נתן'),
```

Comment:
The break here will make it harder to translate things when it come to other grammars.
Make it a single string and remove the fstrings.

The same applies for the rest of the project.